platform_plugin_superset/superset_xblock.py

```diff
@@ -0,0 +1,331 @@
+            .replace("'", "")
+            .replace('"', "")
+            .replace("[", "")
+            .replace("]", "")
+            .strip()
+        )
+        context = {
+            "display_name": self.display_name,
+            "superset_internal_url": self.superset_internal_url,
+            "superset_url": self.superset_url,
+            "superset_username": self.superset_username,
+            "superset_password": self.superset_password,
+            "dashboard_uuid": self.dashboard_uuid,
+            "filters": filters,
+            "display_name_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "display_name"
+            ],
+            "superset_internal_url_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "superset_internal_url"
+            ],
+            "superset_url_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "superset_url"
+            ],
+            "superset_username_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "superset_username"
+            ],
+            "superset_password_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "superset_password"
+            ],
+            "dashboard_uuid_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "dashboard_uuid"
+            ],
+            "filters_field": self.fields[  # pylint: disable=unsubscriptable-object
+                "filters"
+            ],
+        }
+
+        frag = Fragment()
+        frag.add_content(
+            self.render_template("static/html/superset_edit.html", context)
+        )
+        frag.add_css(self.resource_string("static/css/superset.css"))
+
+        # Add i18n js
+        statici18n_js_url = self._get_statici18n_js_url()
+        if statici18n_js_url:
+            frag.add_javascript_url(
+                self.runtime.local_resource_url(self, statici18n_js_url)
+            )
+
+        frag.add_javascript(self.resource_string("static/js/superset_edit.js"))
+        frag.initialize_js("SupersetXBlock")
+        return frag
+
+    @XBlock.json_handler
+    def studio_submit(self, data, suffix=""):  # pylint: disable=unused-argument
+        """
+        Save studio updates.
+        """
+        self.display_name = data.get("display_name")
+        self.superset_internal_url = data.get("superset_internal_url")
+        self.superset_url = data.get("superset_url")
+        self.superset_username = data.get("superset_username")
+        self.superset_password = data.get("superset_password")
+        self.dashboard_uuid = data.get("dashboard_uuid")
+        filters = data.get("filters")
+        self.filters = []
+        if filters:
+            for rlsf in filters.split(","):
+                rlsf = rlsf.strip()
+                try:
+                    field, value = rlsf.split("=")
+                    self.filters.append(f"{field}='{value}'")
```

Comment:
This is limited to filters of type `org='edX'`. All available SQL operators should be supported here:

`IN, !=, LIKE,` etc

As this is the initial version, this is sufficient enough to meet the requirements and we can go ahead with this restriction and improve it in a following PR. @mariajgrimaldi 

Or do you have some ideas for this input field?

I was trying to avoid user error by missing commas. But we can also avoid it by just let the user input everything